src/utilities/command-definition-utils.ts

```diff
@@ -0,0 +1,95 @@
+import { CommandDefinition } from "../interfaces/command-definition";
```

Comment:
👏 what a tidy file. was very pleasant to read - seriously, nice work

Reply:
Thank you! The pairing session we had was great - a second set of eyes was super helpful to see what could be broken out as separate responsibilities